### DISCUSSION TITLE
Can we use pothos schema builder as our gateway for federation?

### DISCUSSION THREAD

**Discussion by wakywayne** [hayes/pothos#779] [Q&A]
I know the docs show how to use pothos as a subgraph for a federation schema, but would there need to be any special configuration in order to use pothos to build my gateway schema? Or is this just not recommended?

**Comment by hayes** [comment]
I'm not sure I understand the use case here.  Generally the gateway schema is generated dynamically from the schemas of your sub-graphs.  Pothos is meant more for building schemas manually.  The existing federation plugin is meant for building sub-graphs, and not intended for the gateway.  I've only used apollos gateway, but it the sub-graphs should work with any gateway.
